test/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.t.sol

```diff
@@ -1467,12 +1669,22 @@ contract BancorVirtualSupplyBondingCurveFundingManagerTest is ModuleTest {
         );
         // Static price has increased after buy
         assertGt(staticPriceAfterBuy, staticPriceBeforeBuy);
+        /*
+        Question:Unsure about how to proceed here. If the supply and collateral tokens have a big difference in decimals (i.e. 18 and 6) and we start with virtual 
+        supply/collateral of 1/1, the first buy will move the price by a lot, since we will be effectively depositing 1e12 collateral tokens (1 + padding to 18). 
+        This will exceed the delta by a lot, but it IS expected behavior.
+        If we set the initial supply to 1e18 and the initial collateral t 1e12 (what we can call "realistic values"), the delta holds again. But on the ohter hand, we
+        aren't testing on the empty curve, just a "normal" point in it. 
```

Comment:
Personally I would go with testing for realistic values. The BCRG is also building a simulation tool which will also (I think) help with defining the edge cases which we should then warn the user for